arm-relay/2017-04-01/swagger/relay.json

```diff
@@ -1575,9 +1581,6 @@
           "description": "Resource tags"
         }
       },
-      "required": [
```

Comment:
If we keep the Required in the tracked resource, than for the patch operation we cant use the same Model as the location is required. can you please suggest how to implement it so that we can use the same model for PUT and PATCH.